registry/domain/services/organization_domain_service.py

```diff
@@ -15,6 +15,20 @@ def __init__(self):
         self.blockchain_util = BlockChainUtil(provider_type="HTTP_PROVIDER",
                                               provider=NETWORKS[NETWORK_ID]['http_provider'])
 
+    def organization_exist_in_blockchain(self, org_id):
+        method_name = "getOrganizationById"
+        contract = self.blockchain_util.load_contract(path=TEST_REG_CNTRCT_PATH)
+        contract_address = self.blockchain_util.read_contract_address(net_id=NETWORK_ID, path=TEST_REG_ADDR_PATH,
+                                                                      key='address')
+        positional_inputs = (web3.Web3.toHex(text=org_id),)
+        contract = self.blockchain_util.contract_instance(contract_abi=contract, address=contract_address)
+
+        org_data = self.blockchain_util.call_contract_function(contract=contract, contract_function=method_name,
+                                                               positional_inputs=positional_inputs)
+        logger.info(f"Org data :: {org_data}")
+        org_found = org_data[0]
+        return org_found
```

Comment:
All this functions are there in blockchian_util we can use that

Reply:
we are using it from the blockchain_utils